template/header.mustache

```diff
@@ -131,6 +131,7 @@
                         <a href='?{{{forms.get.page}}}={{{forms.pages.login}}}' class='loginButtonLink'>
                             <div class='mediaButton loginButton'><img src='{{{page.template_path}}}images/user.png' class='loginButtonImage' /> Log In</div>
                         </a>
+					<a href="?page=register" class="header-register"><button class="mediabutton btn">Register</button></a>
```

Comment:
For get parameters and page values we use mustache constants, this makes it possible to do solution-wide changes and abstracts away the rigidity of URL requests:
`?{{{forms.get.page}}}={{{forms.pages.register}}}`

The parameters that can be used in this way are defined here: https://github.com/OneHourGameJam/OneHourGameJam/blob/e1ecda9a7389ef8c8262312f7b6606c5750f13b8/php/presenters/FormPresenter.php
so for example `$formViewModel->get["page"] = GET_PAGE;` translates to `{{forms.get.page}} and `$formViewModel->form["deletejam"]["jam_id"] = FORM_DELETEJAM_JAM_ID;` translates to `{{forms.form.deletejam.jam_id}}` etc